server/routes/stock-details.js

```diff
@@ -20,49 +21,15 @@ const defaultDetails = {
     fifty_two_week_high: 0
 }
 
-const stockInfo = [
-    {
-        company_name: "Tesla",
-        ticker: "TSLA",
-        current_price: 615.18,
-        previous_close: 570.15,
-        open: 602.14,
-        last_price: 609.89,
-        change: -0.23,
-        change_percent: "-0.04%",
-        currency: "USD",
-        volume: "16.205M",
-        pe_ratio: "585",
-        shares_owned: 553,
-        avg_vol: "31.399M",
-        market_cap: "587.525B",
-        fifty_two_week_low: 321.24,
-        fifty_two_week_high: 900.84
-    },
-    {
-        company_name: "AMC",
-        ticker: "AMC",
-        current_price: 49.11,
-        last_price: 49.34,
-        previous_close: 49.15,
-        open: 52.14,
-        change: -5.71,
-        change_percent: "-10.37%",
-        currency: "USD",
-        volume: "138M",
-        pe_ratio: "N/A",
-        shares_owned: 0,
-        avg_vol: "141M",
-        market_cap: "25.33B",
-        fifty_two_week_low: 2.54,
-        fifty_two_week_high: 72.19
-    }
-]
 
 /* GET stock details. */
 router.get('/:ticker', function (req, res) {
-    const details = stockInfo.filter((stock) => stock.ticker === req.params.ticker);
-    res.send(details.length === 1 ? details[0] : defaultDetails)
+    stockInfo.find().then(stockList => {
+        const details = stockList.filter((stock) => stock.ticker === req.params.ticker);
```

Comment:
instead of using find() with an empty query, could we query with req.params.ticker instead, so we don't request for the whole stocklist? would save us network weight